src/utilities.cpp

```diff
@@ -327,3 +327,17 @@ void now() {
     auto nanos = sc::duration_cast<sc::nanoseconds>(now.time_since_epoch()).count();
     Rcpp::Rcout << nanos << std::endl;
 }
+
+
+template <typename D>
+using time_point = std::chrono::time_point<std::chrono::system_clock, D>;
+using seconds = std::chrono::duration<std::int_fast64_t>;
+int _RcppCCTZ_getOffset(long long s, const char* tzstr) {
+    // reloading the time zone at each point is inefficient, probably
+    // want to try to cache some of this:
+    cctz::time_zone tz;
+    load_time_zone(tzstr, &tz);
+    const auto tp = time_point<seconds>(seconds(s));
+    auto abs_lookup = tz.lookup(tp);
+    return abs_lookup.offset;
+}
```

Comment:
Nice work.

Reply:
Thanks; I have added a NEWS and ChangeLog and also bumped the minor version number from 0.2.6 to 0.2.7.

Reply:
Test is a little difficult, as we can't directly call this function from R, so we'd need a C++ only test. Not at all sure how to do this; shall we leave it as-is right now and look into this when we add additional C low-level routines?

Reply:
Good point about the test (esp from another package) -- that is doable but not worth it.

But can we test the basic offset computation?  Maybe from a wrapper?

Reply:
I think the only way we can test the call from R is if we make an R function for it or for a function that calls it (which is for example what `nanotime` does. Then we would have to document it at the R level, right? Or can we write a wrapper without exporting it at the package level (which we could call something like `_RcppCCTZ_test_exported_C_functions` and would simply return TRUE or FALSE)?

Reply:
You probably have thought much _much_ harder about this particular question than I have.  

So if we can't, we can't.  The tests can of course come from the `nanotime` side too.

Reply:
`nanotime` makes that call and tests it with a few offsets, so I think it's OK for this function. That said, I'm not sure what's the best way to handle this issue that will come up again if additional functions are exported. Will think about that more...

Reply:
In Rcpp we went balistic (as we had to ...) and within the unit testing framework build and a run a client package.  It's all just scripting ...

Reply:
Seems the way to go: will look at how it's done in Rccp.

Reply:
Not sure we need to here.  Tests over from nanotime seems fine and legit given that we own both ends.